app/ui-react/packages/api/src/useIntegrationHelpers.tsx

```diff
@@ -245,6 +245,16 @@ export const useIntegrationHelpers = () => {
     }
   };
 
+  const downloadSupportData = async (data: any) => {
+    const body = await callFetch({
+      body: data,
+      headers: apiContext.headers,
+      method: 'POST',
+      url: `${apiContext.apiUri}/support/downloadSupportZip`,
+    });
+    saveAs(await body.blob(), 'syndesis.zip');
```

Comment:
This doesn't seem to be working for me on firefox at least, I don't see it triggering a download even though it looks like the backend request completes successfully.  Have a look at the `exportIntegration` function in this same file, that one works.

Reply:
Interesting, I'm seeing the download work in FF `60.6.2esr` although when inspecting the network traffic for that api call, it looks like no headers are being sent with the request. 

Thx for pointing to a working example, I'll keep digging and see if I can find out what's going on.

Reply:
So I didn't find anything strange and kinda stumped on this. I just pushed some updates, could you maybe try again and see if it still does it? I was seeing some unrelated errors in the console in firefox though, wasn't sure if it was just this PR or on master also. Gonna check it out now. 

You had a good point offline about the storybook env possibly having some impact, maybe do a build locally and see if it works then? What OS and FF version? Lemme know, we'll get it figured out.

Reply:
Oh and when I tried to use `exportIntegration` as a guide I ran into errors, can't remember exactly what they were but I knew it would take some investigating and since I was seeing the downloads work, I figured it would be good to touch base again before spending a lot of time trying to debug why that didn't work.